ansible_image_validation/configurations/set-env-configs.sh

```diff
@@ -3,9 +3,9 @@ export TABLE_NAME="imagevalidationstatus"
 export CONTAINER_NAME="imagevalidations"
 export ACCOUNT_NAME="pbasnalimagestore"
 export STORAGE_ACCOUNT_RESOURCE_GROUP="pbasnal-rg"
-export MAX_VM_TO_VALIDATE=5
-export VALIDATION_PERIOD=4
-export RESOURCE_GROUP="pbasnal-rg2"
+export MAX_VM_TO_VALIDATE=2
+export VALIDATION_PERIOD=1
+export RESOURCE_GROUP="anujmaurya-imagevalidation-rg"
```

Comment:
May be Resource Group => can be variable name instead of hardcoding

Reply:
This file creates the env variables. It's just a helper file which sets up the environment variables for execution in the following scripts. Mostly for local testing.